util/lib_posix.cc

```diff
@@ -0,0 +1,614 @@
+//  Copyright (c) 2013, Facebook, Inc.  All rights reserved.
```

Comment:
Good idea! I'll change that and commit again later today after the manual pass to make sure I did not revert the last changes.